mockserver/data_reader.go

```diff
@@ -28,12 +28,73 @@ type BlockHeader struct {
 	NextBlock chainhash.Hash `json:"nextblockhash"`
 }
 
+type Transaction struct {
+	TxId          chainhash.Hash `json:"txid"`
+	Hash          chainhash.Hash `json:"hash"`
+	Version       int32          `json:"version"`
+	Size          int32          `json:"size"`
+	VSize         int32          `json:"vsize"`
+	Weight        int32          `json:"weight"`
+	LockTime      uint32         `json:"locktime"`
```

Comment:
please add a comment explaining this type

Reply:
Type removed.